conference/src/main/java/com/kyriba/conference/management/dao/PresentationRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.kyriba.conference.management.dao;
+
+import com.kyriba.conference.management.domain.Presentation;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.time.LocalTime;
+import java.util.stream.Stream;
+
+
+public interface PresentationRepository extends JpaRepository<Presentation, Long>
```

Comment:
You don't need JpaReporsitory here

Reply:
CrudRepository: Iterable<T> findAll();
JpaReporsitory: List<T> findAll(); - that is more convenient to avoid spliterator in ScheduleController